src/Microsoft.ML.Data/DataLoadSave/Binary/CodecFactory.cs

```diff
@@ -44,13 +44,13 @@ public CodecFactory(IHostEnvironment env, MemoryStreamPool memPool = null)
             _loadNameToCodecCreator = new Dictionary<string, GetCodecFromStreamDelegate>();
             _simpleCodecTypeMap = new Dictionary<DataKind, IValueCodec>();
             // Register the current codecs.
-            RegisterSimpleCodec(new UnsafeTypeCodec<DvInt1>(this));
+            RegisterSimpleCodec(new UnsafeTypeCodec<sbyte>(this));
             RegisterSimpleCodec(new UnsafeTypeCodec<byte>(this));
-            RegisterSimpleCodec(new UnsafeTypeCodec<DvInt2>(this));
+            RegisterSimpleCodec(new UnsafeTypeCodec<Int16>(this));
```

Comment:
>RegisterSimpleCodec(new UnsafeTypeCodec<Int16>(this)); [](start = 12, length = 54)

We will want the nullable types to be serializable.

Reply:
This is not needed since there will no nullable types 👍

Reply:
Good, thanks @codemzs. Incidentally were you going to update the description of #673 pursuant to our discussions yesterday? It is still the old description.

---
In reply to: [210744044](https://github.com/dotnet/machinelearning/pull/683#discussion_r210744044) [](ancestors = 210744044)